packages/react/src/Section/Section.tsx

```diff
@@ -0,0 +1,175 @@
+import React, {forwardRef, PropsWithChildren, useCallback, useMemo} from 'react'
+import clsx from 'clsx'
+import type {BaseProps} from '../component-helpers'
+
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/section/section.css'
+import styles from './Section.module.css'
+
+export const SectionPaddingVariants = ['none', 'condensed', 'normal', 'spacious'] as const
```

Comment:
Is there a plan to also support the full spacing scale here?

Reply:
No plans to support the full spacing scale. This component is designed to be intentionally restrictive on the spacing that can be applied to sections.